apps/extension/webpack/webpack.common.js

```diff
@@ -70,13 +74,40 @@ export default {
     },
   },
   plugins: [
+    new webpack.CleanPlugin(),
```

Comment:
What does this do? can you add a code comment?

Reply:
previously, webpack left all existing content in dist while building. mostly this worked as new assets clobbered old assets, but this kind of thing commonly leads to issues with old content from old builds persisting in the new builds.